circuitbreaker.py

```diff
@@ -61,7 +61,7 @@ def call(self, func, *args, **kwargs):
         """
         if self.opened:
             if self.fallback_function:
-                self.fallback_function()
+                self.fallback_function(args, kwargs)
```

Comment:
Please pass the arguments and keyword arguments as those to the fallback function (as in line 68).
This then still allows the actual fallback function to group them into two simple lists if needed (as in line 56)
```suggestion
                self.fallback_function(*args, **kwargs)
```

Reply:
Yes, you are right I did not realize, will fix

Reply:
Great, thanks!
Now we only need the tests passing in Travis and we can merge it!